sdk/storage/azure-storage-common/src/crypt.cpp

```diff
@@ -15,91 +15,59 @@
 #include <openssl/sha.h>
 #endif
 
+#include <algorithm>
 #include <stdexcept>
 #include <vector>
 
+#include "azure/core/http/http.hpp"
 #include "azure/storage/common/storage_common.hpp"
 
 namespace Azure { namespace Storage {
 
   namespace Details {
-    static const char* c_unreserved
-        = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-._~";
     static const char* c_subdelimiters = "!$&'()*+,;=";
```

Comment:
This is a public static in the namespace (yes I know its in Details).  It should start with a Capital
```suggestion
    static const char* SubDelimiters = "!$&'()*+,;=";
```

Reply:
Hi @RickWinter , we talked about this in 10/23 meeting, and according to the meeting minutes, we didn't get any conclusion yet.

Reply:
seems there's already an issue https://github.com/Azure/azure-sdk-for-cpp/issues/898 tracking this. I'm going to make the change together with other constant vriables.